engine/src/test/scala/cromwell/engine/workflow/BackendCallKeySpec.scala

```diff
@@ -18,15 +18,15 @@ class BackendCallKeySpec extends FlatSpec with Matchers with Mockito {
     wd.fileSystems returns defaultFileSystems
     wd.workflowRootPathWithBaseRoot(anyString) answers { root => s"$root/wfName/uuid".toPath(defaultFileSystems) }
 
-    new BackendCallKey(call, None, 1).callRootPathWithBaseRoot(wd, root).toString shouldBe "/root/wfName/uuid/call-callName"
-    new BackendCallKey(call, Some(0), 1).callRootPathWithBaseRoot(wd, root).toString shouldBe "/root/wfName/uuid/call-callName/shard-0"
-    new BackendCallKey(call, Some(0), 2).callRootPathWithBaseRoot(wd, root).toString shouldBe "/root/wfName/uuid/call-callName/shard-0/attempt-2"
-    new BackendCallKey(call, None, 2).callRootPathWithBaseRoot(wd, root).toString shouldBe "/root/wfName/uuid/call-callName/attempt-2"
+    Backend.callRootPathWithBaseRoot(new BackendCallKey(call, None, 1), wd, root).toString shouldBe "/root/wfName/uuid/call-callName"
+    Backend.callRootPathWithBaseRoot(new BackendCallKey(call, Some(0), 1), wd, root).toString shouldBe "/root/wfName/uuid/call-callName/shard-0"
+    Backend.callRootPathWithBaseRoot(new BackendCallKey(call, Some(0), 2), wd, root).toString shouldBe "/root/wfName/uuid/call-callName/shard-0/attempt-2"
+    Backend.callRootPathWithBaseRoot(new BackendCallKey(call, None, 2), wd, root).toString shouldBe "/root/wfName/uuid/call-callName/attempt-2"
```

Comment:
I moved the code that creates the path for a call directory from the key to the Backend. I remember people being surprised that this was in the key class. It was also probably trying to remove a dependency even though I admit I don't remember which one specifically -_-